feature/appdetail/src/androidTest/java/com.merxury.blocker.feature.appdetail/AppDetailScreenTest.kt

```diff
@@ -136,7 +136,7 @@ class AppDetailScreenTest {
                 AppDetailScreen(
                     appInfoUiState = AppInfoUiState.Success(
                         appInfo = appInfoTestData,
-                        appIcon = null,
+                        iconBasedTheming = null,
```

Comment:
the original name `icon` should be fine I think.

Reply:
It's better to understand this parameter's function.
iconBasedTheming may be null, but appIcon isn't null.